.circleci/config.yml

```diff
@@ -252,8 +252,9 @@ workflows:
               only: /.*/
       - build_linux_debug
       - build_macos_debug
+  documentation:
+    jobs:
       - generate_doc:
           filters:
-            branches:
-              only:
-                - develop
+            tags:
+              only: /^v.*/
```

Comment:
You have to add this if you want to run on tags only
```
branches:
    ignore: /.*/
```

Reply:
Thanks! Resolved!